.github/workflows/flutter_ci.yml

```diff
@@ -16,8 +16,11 @@ jobs:
       - uses: subosito/flutter-action@v2
         with:
           cache: true
+      - run: flutter pub get
+      - name: melos bootstrap
+        run: dart run melos bootstrap
```

Comment:
This whole block can be replaced with `https://github.com/bluefireteam/melos-action` and then we can run melos commands like `melos format --set-exit-if-changed .`.